src/tools/plotter/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(iDynTreePlotter)
+
+find_package(Qt5 COMPONENTS Widgets Charts)
+if (${Qt5_FOUND})
```

Comment:
It took some time to me to understand what you mean here by `Qt5_FOUND` . 
I suggest to move this to a `Qt5Charts_FOUND` , and eventually add `Charts` as an `OPTIONAL_DEPENDENCY` in iDynTreeDependencies.cmake